### PR TITLE
fix searching help center

### PR DESCRIPTION
**Pull request by eduard-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/help-centre/components/_search-section.tsx [hunk 1]
```diff
@@ -96,8 +97,22 @@ const SearchSection = () => {
         setSearchValue(sanitize(e.target.value))
     }
 
+    const new_all_questions: TNewQuestionsData[] = React.useMemo(() => all_questions.map((section) => {
+        return {
+            ...section,
+            category: localize(section.category),
+            questions: section.questions.map((q) => {
+                return {
+                    ...q,
```
Comment (habib-deriv) [inline src/pages/help-centre/components/_search-section.tsx:106]: The variable names **section**, and **q** are not very descriptive and may be difficult to understand for someone unfamiliar with the code. Consider using more descriptive names like section_data, and question 
```suggestion
                    ...q,
```

Comment (habib-deriv) [inline src/pages/help-centre/components/_search-section.tsx:106]: The variable names section, and q are not very descriptive and may be difficult to understand for someone unfamiliar with the code. Consider using more descriptive names like section_data, and question

**Review by sergei-deriv — DISMISSED** [review]

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Review by niloofar-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**